compiler/rustc_trait_selection/src/traits/select/mod.rs

```diff
@@ -937,7 +937,7 @@ impl<'cx, 'tcx> SelectionContext<'cx, 'tcx> {
                                     .infcx
                                     .at(&obligation.cause, obligation.param_env)
                                     .trace(c1, c2)
-                                    .eq(DefineOpaqueTypes::No, a.args, b.args)
+                                    .eq(DefineOpaqueTypes::Yes, a.args, b.args)
```

Comment:
All of the `::Yes` in this file need to have a FIXME, probably opened with a new PR...